client/gutenberg/extensions/map/editor.scss

```diff
@@ -6,3 +6,12 @@
 		margin-right: 0.4em;
 	}
 }
+.components-text-control-api-key {
```

Comment:
It might be good to prefix these classes with jetpack_map_block

Reply:
Good call: https://github.com/Automattic/wp-calypso/pull/28730/commits/b3d41c548766d62f66be33258464999447a3bc7d